src/components/Input.tsx

```diff
@@ -5,30 +5,51 @@ const Input = ({
   placeholder,
   helpText,
   disabled,
+  className,
+  value,
+  onChange,
+  mandatoryField,
+  errorMessage,
+  optional = false,
 }: {
   type: string;
   name: string;
   label: string;
   placeholder?: string;
   helpText?: string;
   disabled?: boolean;
+  className?: string;
+  value?: string;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  mandatoryField?: boolean;
+  errorMessage?: string;
+  optional?: boolean;
```

Comment:
No. `required` and `optional` are two different props. `optional` indicates that this field is optional and Required indicates that this field is mandatory. Optional props were needed because optional is written in front of some fields that this field is optional.